cmd/operator/main.go

```diff
@@ -66,6 +66,7 @@ func main() {
 		WatchdogInterval:          0,
 		WatchdogMaxRestartsPerMin: maxPodRestartsPerMinute,
 		WatchNamespace:            watchNamespace,
+		ServiceReconcilerEnabled:  !disableServiceReconciler,
```

Comment:
is it possible to add a unit test or e2e test for this?

Reply:
It is not, actually.... It would require creating an ad-hoc Dapr cluster so it doesn't fit with our E2E test framework.